src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$transition-duration: 300ms;
+$link-hover-color: #00acdc;
+$card-hover-text-color: #34568b;
```

Comment:
These colours can be used for other purposes too, so better to name them as it looks like (grey, blue, etc.) or as what they used for (text-main, text-secondary, etc.)